README.md

```diff
@@ -263,47 +261,45 @@ skate('my-element', {
       // This defaults to `false`. If you specify `true` the property name is
       // dash-cased and used as the attribute it should keep in sync with. If
       // you specify a `string`, it is used as the attribute name.
-      attr: false,
+      attribute: false,
 
-      // This property should be notified of changes when these dependencies
-      // are notified of changes.
-      //
-      // You may also specify dependencies as space-separated string.
-      deps: [
-        // This property's notify events will be triggered when its sibling
-        // `dependencyProperty` is notified of changes.
-        'dependencyProperty',
-
-        // This property's notify events will be triggered when the descendant's
-        // `deepDependencyProperty` notify events are triggered.
-        //
-        // At the time the dependency event bubbles up from the descendant, it
-        // must be accessible via `this.some.nested.child`. If it is not, then
-        // the dependency is ignored.
-        'some.nested.child.deepDependencyProperty'
-      ],
+      // Serializes the property value when it is set so that it can be stored
+      // as an attribute value. This only is called if this property is linked
+      // to an attribute.
+      serializeToAttribute: function () {},
 
-      // Custom getter. The return value is used as the property value when
-      // retrieved. If you don't specify a getter, the value that it was set as
-      // is returned regardless of if you've specified a setter.
-      //
-      // To make a property "readonly", specify a getter without a setter.
-      get: function () {},
+      // Deserializes the attribute value when it is set so that it can be
+      // set as the property value. This only is called if this property is
+      // linked to an attribute.
+      deserializeFromAttribute: function () {},
+
+      // This will be used as the default value for the property. If you specify
+      // a function then it will be invoked and the return value will be used.
+      // This option will also be used in place of the value returned from the
+      // `get()` option if it returns `undefined`.
+      default: 'initial value'
 
       // Whether or not to trigger events when the property changes. Defaults to
-      // `true`. If you do not want events triggered, set this to a falsy value.
+      // `false`. If you do not want events triggered, set this to a falsy value.
```

Comment:
> If you do not want events triggered, set this to a falsy value.

I guess should talk about if you want to enable it, since the default has been inverted.